python2verilog/ir/context.py

```diff
@@ -271,7 +279,7 @@ def add_global_var(self, var: Var):
         var = typed_strict(var, Var)
         if (
             var in self._global_vars
-            or var in self._input_vars
+            or var in self.input_vars
             or var in self.output_vars
```

Comment:
I would double check this logic to make sure you're accessing the correct variables